variables.tf

```
@@ -31,7 +31,7 @@ variable "ip_ranges" {
   EOT
   type = object({
     private_primary    = string
-    private_k8s        = list(object({ name = string, pods = string, svcs = string }))
+    private_k8s        = list(object({ pods_name = string, svcs_name = string, pods = string, svcs = string }))
```

Being more verbose about what the attributes are for
```suggestion
    private_k8s        = list(object({ pods_cidr = string, pods_rname = string, svcs_cidr = string, svcs_rname = string }))
```
For `pods_cidr` instead of just `pods` to avoid ambiguity.

For `pods_rname` (as in "range name") instead of `pods_name` as we're naming the _range_ here instead of the pods